src/BlazorIdentity.UI/Login.razor

```diff
@@ -1,80 +1,112 @@
 ﻿@typeparam TUser where TUser : class
 
+@using Microsoft.Extensions.Logging
 @inject IBlazorSignInManager<TUser> SignInManager
+@inject IBlazorUserManager<TUser> UserManager
+@inject ILogger<Login<TUser>> logger
+@inject NavigationManager NavManager
 
 <EditForm class="col-5" EditContext="editContext" OnSubmit="@HandleSubmit">
-    <DataAnnotationsValidator />
-    <ValidationSummary />
-
-    <div class="mb-3">
-        <label for="email" class="form-label">Email address</label>
-        <InputText id="email" class="form-control" @bind-Value="loginModel.Email" />
-    </div>
-    <div class="mb-3">
-        <label for="password" class="form-label">Password</label>
-        <InputText id="password" type="password" class="form-control" @bind-Value="loginModel.Password" />
-    </div>
-    <div class="mb-3 form-check">
-        <InputCheckbox id="rememberMe" class="form-check-input" @bind-Value="loginModel.RememberMe" />
-        <label class="form-check-label" for="exampleCheck1">Remember me</label>
-    </div>
-    <div>
-        <button type="submit" class="btn btn-primary">Login</button>
-    </div>
+	<DataAnnotationsValidator />
+	<ValidationSummary />
+
+	<div class="mb-3">
+		<label for="email" class="form-label">Email address</label>
+		<InputText id="email" class="form-control" @bind-Value="loginModel.Email" />
+	</div>
+	<div class="mb-3">
+		<label for="password" class="form-label">Password</label>
+		<InputText id="password" type="password" class="form-control" @bind-Value="loginModel.Password" />
+	</div>
+	<div class="mb-3 form-check">
+		<InputCheckbox id="rememberMe" class="form-check-input" @bind-Value="loginModel.RememberMe" />
+		<label class="form-check-label" for="exampleCheck1">Remember me</label>
+	</div>
+	<div>
+		<button type="submit" class="btn btn-primary">Login</button>
+	</div>
+	<br />
+	<div>
+		<p>
+			<a href="/Account/ForgotPassword">Forgot your password?</a>
+		</p>
+		<p>
+			<a href="/Account/Register/@ReturnUrl">Register as a new user</a>
+		</p>
+		<p>
+			<a href="/Account/ResendEmailConfirmation">Resend email confirmation</a>
+		</p>
+	</div>
+
```

Comment:
Seems this is all showing as changed due to you using tabs instead of spaces. Can you ensure all your changes use spaces please?